ScoutSuite/frontend/src/partials/gcp/computeengine.projects.id.zones.id.instances.id/index.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import { Link } from 'react-router-dom';
+
+import { Partial, PartialValue } from '../../../components/Partial';
+import {
+  partialDataShape,
+  formatDate,
+  convertBoolToEnable,
+} from '../../../utils/Partials';
+import { TabsMenu, TabPane } from '../../../components/Tabs';
+import GetResourceName from '../../../components/GetResourceName';
+import { valueOrNone } from '../../../utils/Partials/index';
+
+const renderTags = tags => {
+  if (!tags || tags.length === 0) return <span>None</span>;
+  return (
+    <div className="partial-value detailed-value">
+      <span className="label">Tags</span>
+      <ul>
+        {Object.entries(tags).map(([key, value]) => (
+          <li key={key}>
+            {key}
+            <ul>
+              {value &&
+                Array.isArray(value) &&
+                value.map((v, i) => <li key={i}>{v}</li>)}
+              {value && !Array.isArray(value) && <li>{value}</li>}
+              {!value || (value.length === 0 && <li>None</li>)}
+            </ul>
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+const renderNetworkInterfaces = network_interfaces => {
+  if (!network_interfaces) return <span>None</span>;
+
+  return (
+    <ul>
+      {network_interfaces.map((ni, i) => (
+        <li key={i}>
+          {ni.name}
+          <ul>
+            <li>{ni.networkIP}</li>
+            <li>
+              Network:{' '}
+              <Link
+                to={`/services/computeengine/resources/networks/${ni.network_id}`}
```

Comment:
when you link to resources this way, is the `path` query param resolved on its own or you also need to manually add it?

Reply:
The `path` query parameter is not required anymore. I also want to change all of these links to my new GetResourceLink component to avoid having to enter the full path manually.